.travis.yml

```diff
@@ -13,28 +13,63 @@ matrix:
   - python: 2.7
     env: CONDA_ENV=py27-min
   - python: 2.7
+    env: CONDA_ENV=py27-min SKIP_CI=--skip-optional-ci
```

Comment:
We have a limited number of concurrent builds on Travis-CI: only five. So extra optional builds clog up other CI infrastructure and make it slower to run new tests. So I'd strongly prefer to only have one extra build optional if possible. The 15 or so you add here is too much :)